manifests/d/DeepL/DeepL/23.11.2.11449/DeepL.DeepL.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: DeepL.DeepL
+PackageVersion: 23.11.2.11449
+InstallerType: exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: -verysilent
+  SilentWithProgress: -silent
+  InstallLocation: --store-path="<INSTALLPATH>"
+UpgradeBehavior: install
+Protocols:
+- deepl
+ProductCode: https%3a##appdownload.deepl.com#windows#0install#deepl.xml
```

Comment:
I made an error in approving this - should this be a valid URL?

Reply:
This product code is inspected by `wingetcreate.exe` and written to the manifest as is.